README.md

```diff
@@ -41,6 +41,16 @@ After [installing Meteor](https://www.meteor.com/install), clone this repository
 
 Running tests
 ------------
-    npm install -g spacejam
-    
-    spacejam test-packages
+    # one-time setup
+    curl https://install.meteor.com | /bin/sh
+    npm install -g spacejam eslint
+    export PATH="$HOME/.meteor:$PATH"
+    cp settings.json.template setting.json
```

Comment:
I've fixed this by directly committing to master -- this was such a trivial change I didn't feel the need for a PR.